### ISSUE TITLE
508-defect-2 [COGNITION]: COVID Vaccine Trials - Text styled like headings must be marked up as headings

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#18246]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I want to be able to navigate the vaccine trial form by headings. I might be confused by the large "Bitter" font text that is styled like a heading, but is not a semantic heading in the HTML.

## Details

We have several Bitter "headings" that are `<p>` tags styled to look like headings. This is a violation of SC 1.3.1 Info and Relationships. Screen reader uses often navigate by headings, so we want these blocks to appear in the Headings menu and respond to the `h` shortcut key. Screenshots attached below.

## Acceptance Criteria

- [ ] Headings retain current styling
- [ ] 0 axe-core violations
- [ ] Headings are properly announced by screen readers

## Environment

* https://staging.va.gov/coronavirus-research/volunteer/sign-up

## Proposed Solution (if known)

```html
<div class="schemaform-block-header">
- <span>Help us understand your health</span>
+ <h3>Help us understand your health</h3>
</div>
```

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](http://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2021-01-07 at 12 12 01 PM](https://user-images.githubusercontent.com/934879/103929468-fcdb6600-50e2-11eb-954b-4f758805171d.png)

---
![Screen Shot 2021-01-07 at 12 12 08 PM](https://user-images.githubusercontent.com/934879/103929489-02d14700-50e3-11eb-85d6-4d7e81610e59.png)

---
![Screen Shot 2021-01-07 at 12 12 13 PM](https://user-images.githubusercontent.com/934879/103929503-06fd6480-50e3-11eb-9051-0370f63d3d64.png)

**Comment by 1Copenut** [comment]
Verified the text has been changed to H2s. Moving to close.
